website/src/_posts/2019-10-01-beam-2.16.0.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title:  "Apache Beam 2.16.0"
+date:   2019-10-01 00:00:01 -0800
+# Date above corrected but keep the old URL:
+permalink: /blog/2019/10/01/beam-2.16.0.html
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+  - markliu
+
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+We are happy to present the new 2.16.0 release of Beam. This release includes both improvements and new functionality.
+See the [download page]({{ site.baseurl }}/get-started/downloads/#2160-2019-10-01) for this release.<!--more-->
+For more information on changes in 2.15.0, check out the
+[detailed release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12345494).
+
+## Highlights
+
+ * Fully support Python 3.5, 3.6, 3.7. ([BEAM-1251](https://issues.apache.org/jira/browse/BEAM-1251))
```

Comment:
We already announced Python 3.5-3.7 support in 2.14.0, I suggest combining two highlights as follows.

"Customizable Docker container images released and supported by Beam portable runners on Python 2.7, 3.5, 3.6, 3.7."

Reply:
sg. done.